ONSdigital/eq-survey-runner
# Add variant functionality and remove titles / multiple questions

### What is the context of this PR?
The test_variant_questions.json should show the main functionality of
question_variants, but needs to be expanded to cover more scenarios.

This PR deletes quite a lot of code which was there to handle multiple
questions and titles. Currently all of the unit tests / remaining functional tests 
are passing and the schemas should all pass validation.

This PR is going to be a pain to merge with the PR for removal of piping, due to the modification
of most of the schemas. I don't think there's any way around that.

@bameyrick This PR does change a few of the templates, they're not major changes, but wanted to let you know in case it affects your work.

### How to review 
- Check the variant test schemas make sense.
- Ensure I haven't been too over-zealous in deleting code / tests
- Probably a good idea to check I haven't made any visual changes to any of the templating. I think this is one thing the functional tests wouldn't be good at catching.
